packages/commonwealth/client/scripts/controllers/server/profiles.ts

```diff
@@ -121,7 +121,6 @@ class ProfilesController {
                 ? JSON.parse(profileData.data)
                 : {};
               const lastActive = profileData.Address.last_active;
-              const isCouncillor = profileData.Address.is_councillor;
               const isValidator = profileData.Address.is_validator;
```

Comment:
We can kill isValidator too, I assume.

Reply:
Right, I thought we said we were going to do that when we discussed this last week.